### ISSUE TITLE
Fraction of autosized heating design capacity not applied to `AirLoopHVAC:UnitaryHeatCool`

### ISSUE THREAD

**Issue by lymereJ** [NREL/EnergyPlus#10292]
Issue overview
--------------
The defect file includes a 1.5 `Fraction of Autosized Heating Design Capacity` in the `Sizing:System` used to size a `AirLoopHVAC:UnitaryHeatCool` object. The factor does seem to be taken into account to size the heating coil, but not the system heating capacity, see excerpt from the HTML report below. The system capacity (unadjusted by the sizing factor) appears to be used in the simulation, not the heating coil capacity: output variables show that the `Heating Coil Heating Rate` divided by `Unitary System Fan Part Load Ratio` returns the system heating capacity.

![image](https://github.com/NREL/EnergyPlus/assets/5149279/434e5ebf-f72e-4b13-9ac0-65dffa937d6b)

![image](https://github.com/NREL/EnergyPlus/assets/5149279/9cda2411-d32e-4478-ab30-e83a57acf327)

### Details
Some additional details for this issue (if relevant):
 - Platform: Windows
 - Version of EnergyPlus 22.2

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added: [defect_file_2.zip](https://github.com/NREL/EnergyPlus/files/13245065/defect_file_2.zip)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by joseph-robertson** [comment]
@lymereJ Are you able to test that https://github.com/NatLabRockies/EnergyPlus/pull/11431 gives the desired behavior?

**Comment by lymereJ** [comment]
@joseph-robertson - Sure, I'll take a look at it some time soon.

**Comment by joseph-robertson** [comment]
@lymereJ Hang on. I'm now seeing that this issue is already fixed on develop. After a quick search through merged PRs that may have fixed the issue already, I noticed https://github.com/NatLabRockies/EnergyPlus/pull/11367, and you are the author! Looks like you already fixed this issue and didn't close it?

**Comment by lymereJ** [comment]
Ah, yes, it appears so. That PR was mostly about the AFN fix, I didn't even think about this issue. Your PR does include some doc changes so it might still be worth merging that part?

**Comment by joseph-robertson** [comment]
> Ah, yes, it appears so. That PR was mostly about the AFN fix, I didn't even think about this issue. Your PR does include some doc changes so it might still be worth merging that part?

Yes, done here https://github.com/NatLabRockies/EnergyPlus/pull/11431.

I will close this issue on account of the second item described in https://github.com/NatLabRockies/EnergyPlus/pull/11367.

**Comment by joseph-robertson** [comment]
Closed in https://github.com/NatLabRockies/EnergyPlus/pull/11367.
